app/controllers/concerns/api/v2/taxonomies_controller.rb

```diff
@@ -93,8 +93,8 @@ def destroy
   end
 
   # overriding public FindCommon#resource_scope to scope only to user's taxonomies
-  def resource_scope(*args)
-    @resource_scope ||= scope_for(resource_class, args).send("my_#{taxonomies_plural}")
+  def resource_scope(...)
+    @resource_scope ||= scope_for(resource_class, ...).send("my_#{taxonomies_plural}")
```

Comment:
Per https://bugs.ruby-lang.org/issues/16378 and https://github.com/ruby/ruby/commit/27fca66207f2c35f2f44f6a7cbbe6fd153546082 `f(arg, ...)` is invalid syntax on ruby <2.7.3.

```suggestion
  def resource_scope(*args, **kwargs)
    @resource_scope ||= scope_for(resource_class, *args, **kwargs).send("my_#{taxonomies_plural}")
```

Reply:
And if you ask why we care, Ubuntu 20.04 ships with Ruby 2.7.0.